tests/test_torch.py

```diff
@@ -340,6 +345,8 @@ def test_embedding(wandb_init_run):
         wandb.log({"loss": 1})
     assert len(wandb_init_run.history.rows[0]) == 82
 
+
+@pytest.mark.skip
```

Comment:
You need to add a reason to the call to skip or CI will complain.  Also, I would like to know the reason.